pkgs/build-support/php/builders/v2/hooks/composer-vendor-hook.sh

```diff
@@ -96,9 +96,23 @@ composerVendorInstallHook() {
     mkdir -p $out
 
     cp -ar composer.json $(composer config vendor-dir) $out/
+    mapfile -t installer_paths < <(jq -r 'try((.extra."installer-paths") | keys[])' composer.json)
+
+    for installer_path in "${installer_paths[@]}"; do
+        # Remove everything after {$name} placeholder
+        installer_path="${installer_path/\{\$name\}*/}";
+        out_installer_path="$out/${installer_path/\{\$name\}*/}";
+        # Copy the installer path if it exists
+        if [[ -d "$installer_path" ]]; then
+            mkdir -p $(dirname "$out_installer_path")
+            cp -ar "$installer_path" "$out_installer_path"
+            # Strip out the git repositories
+            find $out_installer_path -name .git -type d -prune -print -exec rm -rf {} ";"
```

Comment:
Why would need to do that here? Aren't `.git` directory stripped out already?

Reply:
What is the command that `composer` runs that uses `git` ? Can you show us an example? Maybe it has a flag to strip those?

Reply:
For example, using `--prefer-dist` ? (https://stackoverflow.com/questions/25981292/how-to-make-composer-not-create-a-git-directory-for-a-package)

Reply:
It's a preference but it doesn't guarantee all .git directories get removed if someone builds a project from for example private repositories. I don't think it's a bad thing to have.

Reply:
allrighty let's keep it.